code/datums/elements/footstep.dm

```diff
@@ -86,10 +86,10 @@
 	play_simplestep(source)
 
 ///Wrapper for movement triggered footsteps for human step
-/datum/element/footstep/proc/humanstep_wrapper(mob/living/source)
+/datum/element/footstep/proc/humanstep_wrapper(mob/living/source, old_loc, movement_dir, forced, old_locs)
 	SIGNAL_HANDLER
-
-	play_humanstep(source)
+	if(!forced)
+		play_humanstep(source)
```

Comment:
not sure what else this effects apart from forced movement by tram

Reply:
Add it to desants :^)